src/svgcontext.ts

```diff
@@ -155,8 +155,8 @@ export class SVGContext extends RenderContext {
     this.lineWidth = 1.0;
 
     const defaultFontAttributes = {
-      'font-family': Font.SANS_SERIF,
-      'font-size': Font.SIZE + 'pt',
+      'font-family': Tables.lookupMetric('fontFamily'),
+      'font-size': '10pt',
```

Comment:
It's interesting that we have a default for fontFamily, but not for fontSize.

Maybe there should be a:

```
Tables.lookupMetric('fontSize'),
```

that returns 10pt?

Reply:
Wait.... there is already such a field. Doesn't the default look like:

```
  fontFamily: 'Bravura',
  fontSize: 30,
  fontWeight: 'normal',
  fontStyle: 'normal',
```

So instead of '10pt' shouldn't we do a `Tables.lookupMetric('fontSize')` here?

Reply:
We have to fix some elements with no defaults in CommonMetrics and that expect it to be 10. In the end the default should be 30.

Reply:
That's why I think there needs to be a separation between music notation fonts and text fonts (for showing letters on the score). 

30pt doesn't make as much sense for the text elements (like volta bracket labels). 

Similarly, setting 'Roboto Slab' on the gClef symbol also doesn't make much sense.

Reply:
One idea for this would be to have these methods in `flow.ts`:

```ts
  static setMusicFont(...fontNames: string[]): void {
    CommonMetrics.musicfontFamily = fontNames.join(',');
    CommonMetrics.fontFamily = CommonMetrics.musicfontFamily + ',' + CommonMetrics.textfontFamily;
  }

  static getMusicFont(): string[] {
    return Tables.lookupMetric('musicfontFamily').split(',');
  }

  static setTextFont(...fontNames: string[]): void {
    CommonMetrics.textFontFamily = fontNames.join(',');
    CommonMetrics.fontFamily = CommonMetrics.musicfontFamily + ',' + CommonMetrics.textfontFamily;
  }

  static getTextFont(): string[] {
    return Tables.lookupMetric('textFontFamily').split(',');
  }
```

The basic idea would be to have two separate font families (and default font sizes).

If an Element is only music symbols, it will do a lookupMetric on `musicFontFamily`. If an Element is only text characterse, it will lookupMetric('textFontFamily'). If an Element has both, it will lookupMetric('fontFamily'), which is the `${musicFontFamily},${textFontFamily}`.

p.s., i haven't checked the above code. There's probably bugs with this approach (e.g., the joining with a comma).

Reply:
@ronyeh this separation does not much the SVG approach. In a SVG you can have one default that will be inherited by the elements inside. At these level I think that what makes more sense is to have the fontFamily including all the fonts, SMuFL and non SMuFL. Have a look at the SVGs produced by this branch, the redefinition of fonts in SVGs is much leaner.